libs/utils/square_test.go

```diff
@@ -0,0 +1,59 @@
+package utils
+
+import (
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common/math"
```

Comment:
what is the point here to use go-eth lib?

Reply:
sorry, I think it go auto imported somehow. Fixed now